libraries/cyclestreets-fragments/src/main/res/values/strings.xml

```diff
@@ -63,7 +64,8 @@
 
   <string name="location_editor_hint">Place the mark on the map and then name the location.</string>
   <string name="location_editor_name">Location name :</string>
-  <string name="routenumber_hint">Route number</string>
+  <string name="routenumber_desc">Routes previously planned on CycleStreets can be loaded here, by entering the journey number shown on the webpage or itinerary</string>
+  <string name="routenumber_hint">CycleStreets journey number</string>
```

Comment:
"Journey number" appears to be the naming used both in the itinerary view of our app and also on the website.  Hence the tweak to this hint (as well as prefixing it with "CycleStreets".